build.sbt

```diff
@@ -1,9 +1,22 @@
-enablePlugins(ScalaNativePlugin)
+scalaVersion in ThisBuild := "2.11.12"
 
-name := "cmark-scala"
+nativeLinkStubs := true
 
-version := "0.1.0-SNAPSHOT"
+lazy val root = (project in file("."))
+  .aggregate(cmark, tests)
 
-organization := "tech.sparse"
+lazy val commonSettings = Seq(
+  organization := "tech.sparse",
+  version := "0.1.0-SNAPSHOT"
+)
 
-scalaVersion := "2.11.12"
+lazy val cmark = (project in file("cmark"))
+  .settings(commonSettings)
+  .enablePlugins(ScalaNativePlugin)
+
+lazy val tests = (project in file("tests"))
```

Comment:
I think it is sufficient to only have one module here and use the standard Scala directory structure:

* `src/main/scala` for sources
* `src/test/scala/` for tests